.github/workflows/test.yml

```diff
@@ -5,41 +5,47 @@ on:
       - master
       - "[0-9]+.[0-9]+.x"
   pull_request:
-    types: [opened, synchronize]
```

Comment:
Why remove this? We don't want to run tests for every single branch unless it's beta/master/maintenance and for PRs.

Reply:
The default types are `opened`, `synchronize` and `reopened`, so falling back to the default here will result in pretty much exactly the desired behaviour.

Reply:
I just find it odd to leave the `pull_request` key without any props 😆  In the spirit of being explicit as you were with the other changes, I'd prefer to keep this one as well.

Reply:
It's pretty common to do this in GitHub Actions, for example many of the examples in [the documentation](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows) only specify a trigger without a type. But if you feel strongly about it I will change it.